src/include/duckdb/common/types/value.hpp

```diff
@@ -124,6 +124,10 @@ class Value {
 		throw NotImplementedException("Unimplemented template type for Value::GetValue");
 	}
 	template <class T>
+	T GetValueAsIs() const {
```

Comment:
Rather than creating an additional template function that only works for blobs, we could just directly use `ToStringAsIs`, no? That should clean up the code here as well somewhat.